src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -276,9 +277,7 @@ void QMCDriverNew::makeLocalWalkers(IndexType nwalkers, RealType reserve)
   ScopedTimer local_timer(timers_.create_walkers_timer);
   // ensure nwalkers local walkers in population_
   if (population_.get_walkers().size() == 0)
-  {
-    population_.createWalkers(nwalkers, reserve);
-  }
+    population_.createWalkers(nwalkers, walker_configs_ref_, reserve);
```

Comment:
walker_configs_ref_ was taken as a reference, the state of population_ already gets entangled with the main state walker_configs.

I don't think my modification really changes the fact that createWalkers API needs to take care of existing walker configurations either via internally stored reference or via an argument. I prefer explicit argument.

Reply:
Yes the explicit argument is much better, but I don't like to see redundant arguments (although I've certainly made them). They complicate the logic of the function and you really need two distinct units tests here. 

One for a clean driver statrt i.e. first section where the walker_configs_ref actually carries no information and another with several cases where walker_configs_ref_ does contain walker information and is less than, equal_to, or greater nwalkers or nwalkers+reserve.  I notice that I/we only covered the first test (new driver).  So this already is better since it was easy to see that missing test.

Anyway this doesn't need a change.